src/index.tsx

```diff
@@ -286,7 +294,7 @@ export class App extends React.Component<any, AppState> {
   private onKeyDown = (event: KeyboardEvent) => {
     if (event.key === KEYS.ESCAPE) {
       elements = clearSelection(elements);
-      this.forceUpdate();
+      this.setState({});
```

Comment:
This one might not be needed since we're already  calling setState on the next line